values.yaml

```diff
@@ -634,6 +634,26 @@ keycloak:
   extraVolumeMounts: |
     - name: incore-theme
       mountPath: /opt/bitnami/keycloak/themes/incore-theme
+    
+
+# ----------------------------------------------------------------------
+# APPROVAL APP CONFIGURATION
+# ----------------------------------------------------------------------
+approval:
+  replicas: 1
+  image:
+    repository: approval-app
+    tag: latest
```

Comment:
maybe we can change it to the real value instead of latest?

Reply:
Updated to "master" since incore dev-script repo still uses "master" as the main branch.